### ISSUE TITLE
An error occurs if trying to checkout a pacakge to a directory that already has some contents

### ISSUE THREAD

**Issue by KeithMnemonic** [SUSE/open-build-service-connector#14]
When doing a checkout of of a package such as Apache for example,  if you select a target of /ibs and ii that directory there already exists another package like Documentation you will get an error.

Error running command vscodeObs.obsProject.checkOutProject: directory d:\Data\repos\foo is not empty, the following file already exist: mash-job-doc-gen. This is likely caused by the extension that contributes vscodeObs.obsProject.checkOutProject.

Ideally this should work like osc and you should only get an error if the package directory you are trying to checkout exists. It should not error if there are directories for other packages, but should just create a new directroy with the package name and checkout the files into that directory

**Comment by dcermak** [comment]
Yes, that makes sense and has been a minor gripe for me actually. I'll put this on my roadmap and hopefully get to it soon.
